### PR TITLE
swagger file

### PR DESCRIPTION
**Pull request by ssr8998**
swagger is added
=> contains swagger at /apidocs route
=> authorization api key is added in congif.py file
=> all required models for swagger are created.. but will be useable after creating apis in restx

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import json
 
 # Get configuration from environment
-DATABASE_URI = os.getenv("DATABASE_URI", "postgres://postgres:postgres@localhost:5432/postgres")
+DATABASE_URI = os.getenv("DATABASE_URI", "postgres://postgres:password@localhost:5432/postgres")
```
Comment (joyceluyy) [inline config.py:8]: @ssr8998 Wondering why we want to change the password of the database?

**Review by joyceluyy — COMMENTED** [review]
Seems there are overlapping changes in both branch, restx_suppliers and swagger, not sure which need to be merged first. And the code coverage is dropping below 95%

File: config.py (hunk 1 above)
Comment (ssr8998) [reply config.py:8]: > Seems there are overlapping changes in both branch, restx_suppliers and swagger, not sure which need to be merged first. And the code coverage is dropping below 95%

The swagger is the initial PR and swagger need to be PR first .

Comment (ssr8998) [reply config.py]: DATABASE_URI = os.getenv("DATABASE_URI", "postgres://postgres:postgres@localhost:5432/postgres")
DATABASE_URI = os.getenv("DATABASE_URI", "postgres://postgres:password@localhost:5432/postgres")
was a typo and its fixed .

**Comment by joyceluyy** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/3ba99d8e-579e-4699-82ef-1fdda0bd7729?env_id=ibm:yp:us-south "supplier-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/4bf3eb13-4e76-4496-affe-fa81bd552499 "nyu-supplier-service-sum21") deployed [suppliers](https://github.com/nyusuppliers/suppliers/releases/tag/deploy-ibm-yp-us-south-snr6450-nyu.edu-prod-20210809-023546 "deploy-ibm-yp-us-south-snr6450-nyu.edu-prod-20210809-023546") to [prod](https://console.bluemix.net/apps/2dad892b-58c1-4322-93f9-59f6e4da1ec5?env_id=ibm:yp:us-south "ibm:yp:us-south:snr6450@nyu.edu:prod"), including commit 541049b821c97c9a1af60f0190db33f5fabf3fbb

**Comment by joyceluyy** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/3ba99d8e-579e-4699-82ef-1fdda0bd7729?env_id=ibm:yp:us-south "supplier-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/4bf3eb13-4e76-4496-affe-fa81bd552499 "nyu-supplier-service-sum21") deployed [suppliers](https://github.com/nyusuppliers/suppliers/releases/tag/deploy-ibm-yp-us-south-snr6450-nyu.edu-dev-20210809-024944 "deploy-ibm-yp-us-south-snr6450-nyu.edu-dev-20210809-024944") to [dev](https://console.bluemix.net/apps/2687c3e5-ee27-4331-a2b4-de35932177df?env_id=ibm:yp:us-south "ibm:yp:us-south:snr6450@nyu.edu:dev"), including commit 541049b821c97c9a1af60f0190db33f5fabf3fbb

**Pull request merged**